sql-statements/sql-statement-create-user.md

```diff
@@ -151,6 +153,22 @@ CREATE USER 'newuser9'@'%' PASSWORD EXPIRE;
 Query OK, 1 row affected (0.02 sec)
 ```
 
+创建一个使用资源组 `rg1` 的用户：
+
+```sql
+CREATE USER 'newuser7'@'%' RESOURCE GROUP rg1;
+SELECT USER, HOST, USER_ATTRIBUTES FROM MYSQL.USER WHERE USER='newuser7';
```

Comment:
nit: How about `SELECT USER, HOST, JSON_EXTRACT(USER_ATTRIBUTES, "$.resource_group") from MYSQL.USER;`  so we only output resource group related attributes in the result.

Reply:
@BornChanger  PTAL

Reply:
I think  we can just keep it as it is now.